### PR TITLE
Marketplace search and filter

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add a new page to describe what consumer can search / filter in Marketplace.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | ae59cb6 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-377.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop, @georgiana-caras not all featured are deployed yet but you can start review/comment

**Review by lbadenhop — APPROVED** [review]

**Comment by lbadenhop** [comment]
@cbordier, please check my edits.

**Comment by cbordier** [comment]
@lbadenhop your edits are good to go.

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: Search and filter Marketplace content
+linkTitle: Search and filter Marketplace content
+weight: 5
+---
+
+Learn how you can find content in the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:7]: Learn how you can search through content in the Marketplace.

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## What is indexed and available for searching
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:9]: remove this entire paragraph

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+Search the Marketplace for four types of objects by using their attributes:
+
+* **Category**: title, description
+* **Product**: title, description, documentation (markdown only), category title assigned to the product, product resource specification
+* **Application**: title
+* **Subscription**: product title, product description
+
+## Where to search from
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:18]: Rename to **Search by keyword**

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+The search is based on a match pattern. This mean that the results will have the portion of entered text anywhere in the fields mentioned in previous section.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:20]: Each main page has a search bar that allows to type a keyword and refine the results. 
**Products search** allows you to look for terms in: 

- product title
- product description
- markdown documentation
- product categories
- API specification

From the **Category** page, you can search categories by: 

- title
- description

From the **Applications** page, you can search applications by: 

- Title

From the **Subscriptions** page, you can search subscriptions by: 

- product title
- product description

The search will looks for a partial or complete match with the keyword that is entered in the search bar.

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+### Home page search
+
+Search is available when the home page is activated from the Marketplace settings.
+
+* Search bar from Hero Content (if activated): search with any words matching a product title / description / specification content / Documentation (markdown only) / category title assigned to a product.
+* Browse product: filter the recently published list of products based on product title / description / title assigned to a product.
+  
+### Product search
+
+By default, the product search uses what is entered in the search bar to find a match in product title / description / specification content / Documentation (markdown only) / category title assigned to a product.
```
Comment (georgiana-caras) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:52]: You can use the search bar to find products by keywords with a match in the: 

- product title
- description
- API specification
- Documentation (markdown only)
- categories

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Pull request merged**